spec/models/subscription_spec.rb

```diff
@@ -1,5 +1,45 @@
 require 'rails_helper'
 
 RSpec.describe Subscription, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  it 'validate aniversary day' do
+    company = Fabricate(:company_completed)
+    payment_method = Fabricate(:payment_method)
+    purchaser = Fabricate(:purchaser)
+    plan = Fabricate(:plan)
+
+    new_subscription = Subscription.create!(company: company,
+                                            payment_method: payment_method,
+                                            purchaser: purchaser,
+                                            plan: plan)
+
+    expected_day = Time.zone.today.day > 28 ? 1 : Time.zone.today.day
+
+    expect(new_subscription.aniversary_day).to eq(expected_day)
+  end
+
+  it 'validate aniversary day over day 28' do
+    company = Fabricate(:company_completed)
+    payment_method = Fabricate(:payment_method)
+    purchaser = Fabricate(:purchaser)
+    plan = Fabricate(:plan)
+
+    allow(Date).to receive(:today).and_return(Date.new(2021, 3, 31))
```

Comment:
Uma outra forma legal seria transformar isso em algo dinâmico se quiserem dar uma olhada.

```suggestion
    allow(Date).to receive(:today).and_return(Date.current.end_of_month)
```